example/src/main/assets/push-config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "pushServerURL": "https://testing.zeta.feedhenry.com/api/v2/ag-push/",
+    "android": {
+        "senderID": "847851272667",
+        "variantID": "518fd580-44b8-4a8c-b865-22300384800d",
+        "variantSecret": "a5ada26a-37bd-40f2-8f4a-66f846ffa395"
+    }
+}
```

Comment:
Is there an issue or JIRA to generate this config from mobile-cli?

Reply:
See: https://github.com/aerogear/proposals/pull/31